documentation/airport-guides/ltba.md

```diff
@@ -20,11 +20,11 @@ The airport has 3 runways. The southeastern runway (05/23) is 2580m (8465ft) lon
 * Runway 05 for landing
 * Runway 35L for take-off
 
-**In the simulator the mentioned runway configuration is used.**
+**This also the configuration in use in openScope**
```

Comment:
This _**is**_ also the configuration in use in openScope

Reply:
When you just copy and paste 😄